torch/csrc/jit/tensorexpr/analysis.h

```diff
@@ -46,6 +46,22 @@ class NodeFinder : public IRVisitor {
   std::vector<Node*> nodes;
 };
 
+class VarFinder : public IRVisitor {
+ public:
+  virtual void visit(const Var* v) override {
+    vars.insert(v);
+    IRVisitor::visit(v);
+  }
+
+  static std::unordered_set<const Var*> find(Stmt* s) {
```

Comment:
Is the find() function used somewhere?

Reply:
I use it in a follow up (currently). I'd like to keep it for now.